carbonmark-api/src/app.ts

```diff
@@ -2,9 +2,10 @@ import AutoLoad, { AutoloadPluginOptions } from "@fastify/autoload";
 import * as dotenv from "dotenv";
 import { FastifyPluginAsync } from "fastify";
 import { join } from "path";
+import { LOCAL_ENV_PATH } from "./utils/helpers/utils.constants";
 
 // This is not pretty but we need to reference the env file at the root of the monorepo
-const result = dotenv.config({ path: "../.env.local" });
+const result = dotenv.config({ path: LOCAL_ENV_PATH });
```

Comment:
Nice one and thanks for the extra docs 👍

Could you use `process.cwd` here plz ([example](https://github.com/KlimaDAO/klimadao/pull/1347/files#diff-bc149aa836d2ff3c5bd38465633d65ab9944fa49efa7687204654e8ee247baf5R7))  and remove the constant from utils?

```suggestion
const result = dotenv.config({ path: join(process.cwd(), ".env.local") });
```